ice.c

```diff
@@ -2023,6 +2025,44 @@ static void janus_ice_cb_nice_recv(NiceAgent *agent, guint stream_id, guint comp
 							stream->video_is_keyframe = &janus_h264_is_keyframe;
 					}
 				}
+				/* If we need to handle transport wide cc*/
+				if (stream->do_transport_wide_cc) 
+				{
+					guint16 transport_seq_num;
+					/* Get transport wide seq num */
+					if (janus_rtp_header_extension_parse_transport_wide_cc(buf, buflen, stream->transport_wide_cc_ext_id, &transport_seq_num)==0) {
+						/* Get current timestamp */
+						struct timeval now;
+						gettimeofday(&now,0);
+						
+						/* Create <seq num, time> pair */
+						janus_rtcp_transport_wide_cc_stats* stats = g_malloc0(sizeof(janus_rtcp_transport_wide_cc_stats));
+						if(stats == NULL) {
+							JANUS_LOG(LOG_FATAL, "Memory error!\n");
+							return;
+						}
+						/* Check if we have a sequence wrap */
+						if (transport_seq_num<0x0FFF && (stream->transport_wide_cc_last_seq_num&0xFFFF)>0xF000)
```

Comment:
Can this check be confused by out of sequence packets, and/or retransmissions (we don't do RF4588 yet), thus increasing the cycles more times than it should?